strongbox-security/strongbox-security-api/src/main/java/org/carlspring/strongbox/security/UserAccessModel.java

```diff
@@ -19,6 +21,7 @@
 {
 
     @XmlElement
+    @OneToOne(cascade = CascadeType.ALL)
```

Comment:
Don't think it is necessary since `UserAccessModel` is a field of `org.carlspring.strongbox.security.User` and this class is not a `GenericEntity`. So `UserAccessModel` shouldn't be a `GenericEntity` too, I guess.

Reply:
yeah, maybe
will try it without cascade